spec/requests/notifications_spec.rb

```diff
@@ -44,7 +46,7 @@ def renders_comments_html(comment)
       it "renders the signup page" do
```

Comment:
@rt4914 I'm referring to this line where the describe block says 'it rendered the **signup** page' but in the test it's actually checking that we render the signin page. Hence the mismatch.

Reply:
This has been fixed by updating the text to `renders the signin page`